scripts/convert_FCM_cfg_to_GH_secret.py

```diff
@@ -0,0 +1,21 @@
+import os
+import sys
+from pathlib import Path
+
+
+def convert_config_to_secret(config):
+    s = ""
+    for i in config:
+        s += i.replace("\n", "").replace("\"", "\\\"").replace("{", "\\{").replace("}", "\\}"). \
+            replace("\\n", "\\\\n")
+
+    return s
+
+
+BASE_DIR = Path(os.path.dirname(os.path.abspath(__file__))).parent.absolute()
+
+frame = sys._getframe()
+
+FIREBASE_ADMIN_CONFIG = os.path.join(BASE_DIR, '.firebase_admin_config')
+
+print(convert_config_to_secret(open(FIREBASE_ADMIN_CONFIG).readlines()))
```

Comment:
lol)
@Ivan228777 do we need it?
in my mind we showed out secrets there

Reply:
I don't think that we actually showed our secrets here (at least hope so)), but it is better to remove this print.
This script is needed for converting .firebase_admin_config to format that can be contained in GitHub secrets.

Reply:
okay, I just delete this print